omnibus/Gemfile

```diff
@@ -1,7 +1,7 @@
 source "https://rubygems.org"
 
 gem "omnibus", github: ENV.fetch("OMNIBUS_GITHUB_REPO", "chef/omnibus"), branch: ENV.fetch("OMNIBUS_GITHUB_BRANCH", "main")
-gem "omnibus-software", github: ENV.fetch("OMNIBUS_SOFTWARE_GITHUB_REPO", "chef/omnibus-software"), branch: ENV.fetch("OMNIBUS_SOFTWARE_GITHUB_BRANCH", "main")
+gem "omnibus-software", github: ENV.fetch("OMNIBUS_SOFTWARE_GITHUB_REPO", "chef/omnibus-software"), branch: ENV.fetch("OMNIBUS_SOFTWARE_GITHUB_BRANCH", "CHEF-14521-chef-17-freebsd-13_ruby_patch")
```

Comment:
I'd rather we accomplish this by setting the OMNIBUS_SOFTWARE_GITHUB_BRANCH env var on the bk build rather than override in code.

Reply:
good point, less mucking about that way